metricbeat/module/php_fpm/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: php_fpm
+  title: "php_fpm"
+  description: >
+    PHP-FPM server status metrics collected from PHP-FPM.
+  fields:
```

Comment:
I think I forgot to mention it in the review, but also the metricsets should have a log line in the New(...) part that has a warning that it is experimental. Also see prometheus.